### ISSUE TITLE
Focus gets reset to start of page after tab on Single row action ( clr-dg-action-overflow ) in data grid

### ISSUE THREAD

**Issue by AnjulP21** [vmware/clarity#6305]
## Describe the bug

When using tab on clr-dg-action-overflow in Claritygrid, the focus gets reset to main page once we select any action on the list.

## How to reproduce

It is esily reproducible on ghe clarity page. [https://v4.clarity.design/documentation/datagrid/single-action]

1.  Go to https://v4.clarity.design/documentation/datagrid/single-action
2.  Go to Single row action.
3.  Use tab to traverse to the Single row action button and press enter to open action menu.
4.  Use tab to traverse through the menu an select any menu-item (edit or delete).
5.  After menu-item is selected, try to tab again. Focus gets reshifted to start of the page. 

## Expected behavior

After Single row action menu is closed , Focus should be reset on the Single row action button.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [ ] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [x] v4.x
- [ ] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
_ie: Angular 11_

**Device:**

- Type: [e.g. MacBook]

## Additional notes

_Add any other notes about the problem here._
